wps_tools/__version__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+
+# This information is located in its own file so that it can be loaded
+# without importing the main package when its dependencies are not installed.
+# See: https://packaging.python.org/guides/single-sourcing-package-version
+
+__author__ = """Eric Yvorchuk"""
+__email__ = 'eyvorchuk@uvic.ca'
+__version__ = '0.1.0'
```

Comment:
I don't think this file is necessary. We can define these things in the `setup.py`